rocksdb_admin/admin_handler.cpp

```diff
@@ -497,7 +510,29 @@ AdminHandler::AdminHandler(
   }
 
   if (FLAGS_enable_async_incremental_backup_dbs) {
-    LOG(INFO) << "incremental backup gflag is enabled";
+    db_incremental_backup_thread_ = std::make_unique<std::thread>([this] {
```

Comment:
Lets move this entire thread to be a property/code of backup manager. Ideally we dont have to add so many changes to admin handler.

We can simply add a function there called stop() which will can set the atomic bool and join the thread.